pkgs/development/libraries/qt-5/5.9/qtbase/default.nix

```diff
@@ -110,10 +110,10 @@ stdenv.mkDerivation {
           -e 's#sdk_val=$(/usr/bin/xcrun -sdk $sdk -find $(echo $val | cut -d \x27 \x27 -f 1))##' \
           -e 's#val=$(echo $sdk_val $(echo $val | cut -s -d \x27 \x27 -f 2-))##' \
           ./configure
-      sed -i '3,$d' ./mkspecs/features/mac/default_pre.prf
-      sed -i '27,$d' ./mkspecs/features/mac/default_post.prf
-      sed -i '1,$d' ./mkspecs/features/mac/sdk.prf
-      sed -i 's/QMAKE_LFLAGS_RPATH      = -Wl,-rpath,/QMAKE_LFLAGS_RPATH      =/' ./mkspecs/common/mac.conf
+          substituteInPlace ./mkspecs/common/mac.conf \
+              --replace "/System/Library/Frameworks/OpenGL.framework/" "${darwin.apple_sdk.frameworks.OpenGL}/Library/Frameworks/OpenGL.framework/"
+          substituteInPlace ./mkspecs/common/mac.conf \
+              --replace "/System/Library/Frameworks/AGL.framework/" "${darwin.apple_sdk.frameworks.AGL}/Library/Frameworks/AGL.framework/"
```

Comment:
Why is this not part of the patch file?

Reply:
Can we know the actual ${darwin.apple_sdk.frameworks.AGL} path within patch file?

Reply:
don't think so, but I think replacing paths like this preferable since it doesn't break as easily as a patch

Reply:
Oh thanks for clarifying this. I missed the `${darwin...}` substitutions.

Reply:
btw. these two lines will probably enable `qt5.qtwebkit`, too.

Reply:
Yes, I can build qt5.qtwebkit locally